index.bs

```diff
@@ -419,29 +420,40 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
 
-
 : <dfn>Human Palatability</dfn>
 :: An identifier that is [=human palatability|human-palatable=] is intended to be rememberable and reproducible by typical human
     users, in contrast to identifiers that are, for example, randomly generated sequences of bits [[EduPersonObjectClassSpec]].
 
-
 : <dfn>Public Key Credential Source</dfn>
-:: A [=credential source=] ([[CREDENTIAL-MANAGEMENT-1]]) used by an [=authenticator=] to generate [=authentication assertions=]. A public key credential source has:
+:: A [=credential source=] ([[CREDENTIAL-MANAGEMENT-1]]) used by an [=authenticator=] to generate [=authentication assertions=]. A [=public key credential source=] consists of a [=struct=] with the following [=struct/items=]:
+
+    <dl dfn-for="public key credential source">
+        :   <dfn>type</dfn>
+        ::  whose value is of {{PublicKeyCredentialType}}, defaulting to {{public-key}}.
+
+        :   <dfn>id</dfn>
+        ::  A [=Credential ID=].
+
+        :   <dfn>privateKey</dfn>
+        ::  The [=credential private key=].
 
-    * A [=Credential ID=].
-    * A [=credential private key=].
-    * The [=Relying Party Identifier=] for the [=[RP]=] that created this credential source.
-    * A [=user handle=] for the person who created this credential source.
-    * Optional other information used by the authenticator to inform its UI. For example, this might include the user's
-        {{displayName}}.
+        :   <dfn>rpId</dfn>
+        ::  The [=Relying Party Identifier=], for the [=[RP]=] this [=public key credential source=] is associated with.
+
+        :   <dfn>userHandle</dfn>
+        ::  The [=user handle=] associated when this [=public key credential source=] was created.
```

Comment:
I think we need to point out that this field is nullable.

Reply:
ok.